<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planning Poker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="planningTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>480</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="connectGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QHBoxLayout" name="connectLayout">
     <item>
      <widget class="QLineEdit" name="hostLineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>localhost</string>
       </property>
       <property name="placeholderText">
        <string>Nazwa lub IP serwera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portSpinBox">
       <property name="maximum">
        <number>70000</number>
       </property>
       <property name="value">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Połącz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="chooseGameGroup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QHBoxLayout" name="chooseGameLayout">
     <item>
      <widget class="QPushButton" name="createGameBtn">
       <property name="text">
        <string>Stwórz nową grę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="joinGameBtn">
       <property name="text">
        <string>Dołącz do istniejącej gry</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpisz nazwę serwera oraz port, aby nawiązać połączenie</string>
    </property>
   </widget>
   <widget class="QLabel" name="planningLogo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
